pkg/route/types.go

```diff
@@ -58,6 +59,9 @@ type Config struct {
 	Id           string         `json:"id,omitempty"`           // route ID
 	TenantId     tenant.Id      `json:"tenant,omitempty"`       // TenantId. Derived from URL path. Should not be marshaled
 	UserId       string         `json:"userId,omitempty"`       // user ID / author of route
+	Region       string         `json:"region,omitempty"`       // optional region of route for active-active scenarios - if present, route will only be active in a single region
+	Inactive     bool           `json:"inactive"`               // if true, route will not execute
+	Status       string         `json:"status"`                 // a route running on this instance will have status running, otherwise status will be blank
```

Comment:
For status, why couldn't it be "stopped" instead of just blank?

Reply:
ok, changed it